substrate/network-libp2p/src/traits.rs

```diff
@@ -190,7 +190,7 @@ impl NetworkConfiguration {
 			use_secret: None,
 			min_peers: 25,
 			max_peers: 50,
-			max_incoming_peers_factor: 3,
+			incoming_peers_factor: 3,
```

Comment:
I feel like `3` is too high, especially for bootstrap nodes which probably want 90% of incoming connections.

Reply:
So `2`, probably?

Reply:
And just for information, was choosing `3` because it comes from the [eclipse attack paper](https://www.cs.bu.edu/~goldbe/projects/eclipseEth.pdf).